### PR TITLE
add CLI to create a mosaic from GeoJSON features

### PR DESCRIPTION
**Pull request by vincentsarago**
This PR adds a CLI to create a mosaicJSON from a list of features of featureCollection

### CODE REVIEW

File: cogeo_mosaic/scripts/cli.py [hunk 1]
```diff
@@ -92,6 +93,68 @@ def create(
...
+    quadkey_zoom,
+    min_tile_cover,
+    tile_cover_sort,
+    quiet,
+):
+    """Create mosaic definition file."""
+    mosaicjson = MosaicJSON.from_features(
+        list(features),
+        minzoom,
+        maxzoom,
```
Comment (kylebarron) [inline cogeo_mosaic/scripts/cli.py:143]: Just pass these as keyword args so that you don't need to add more positional arguments to `from_features`

Comment (vincentsarago) [reply cogeo_mosaic/scripts/cli.py:143]: nope because thoses are required in the actual model.

Comment (kylebarron) [reply cogeo_mosaic/scripts/cli.py:143]: Just use this: https://github.com/developmentseed/cogeo-mosaic/pull/63/files#diff-7c1d4d4b7c5cc98334549838d79b4534R109-R136

Don't make them required from the command line, and tell people that if the properties don't exist, an error will be raised.

Comment (vincentsarago) [reply cogeo_mosaic/scripts/cli.py:135]: I don't think it's a good way for now. Happy to review this later

**Pull request merged**